config/config.json

```diff
@@ -100,6 +100,12 @@
     ],
     "include_tables": [],
 
+    "uuid_columns_description": [
```

Comment:
@simondh 
Here you propose a new feature that not related to this PR's title <code>Fix duplicate index names caused by 63 char limit</code>.
If you'd like to propose a new feature - a separate PR should be submitted (good Git practice).

Regarding your possible proposal:
In general, there is a customizable data types configuration <code>nmig/config/data_types_map.json</code>.
If we want to introduce PG-specific types - I would say that something like <code>nmig/config/extra_data_types_map.json</code> needed.
Just as it is with <code>config.json</code> for basic configuration, and <code>extra_config.json</code> for "advanced" configuration.
Besides that, you initialize <code>_uuidColumns</code> in <code>nmig/src/Conversion.ts</code>, but this code doesn't do actual types conversion anyway...